lib/hanami/middleware/content_security_policy_nonce.rb

```diff
@@ -29,10 +29,12 @@ def call(env)
 
         request_nonce = generate_nonce
         env[CONTENT_SECURITY_POLICY_NONCE_REQUEST_KEY] = request_nonce
-        @app.call(env).tap do |response|
-          headers = response[1]
-          headers["Content-Security-Policy"] = sub_nonce(headers["Content-Security-Policy"], request_nonce)
-        end
+
+        _, headers, _ = response = @app.call(env)
```

Comment:
Not such a fan of `_`, but I ack that I like `tap` and `then` maybe a tad too much. 😅